linkis-computation-governance/linkis-entrance/src/main/scala/org/apache/linkis/entrance/interceptor/impl/SQLCodeCheckInterceptor.scala

```diff
@@ -26,8 +26,15 @@ import org.apache.linkis.manager.label.utils.LabelUtil
 class SQLCodeCheckInterceptor extends EntranceInterceptor {
 
   override def apply(jobRequest: JobRequest, logAppender: java.lang.StringBuilder): JobRequest = {
-    val engineType = LabelUtil.getCodeType(jobRequest.getLabels)
-    val languageType = CodeAndRunTypeUtils.getLanguageTypeByCodeType(engineType.toLowerCase())
+    val engineType = {
+      val engineType = LabelUtil.getCodeType(jobRequest.getLabels)
```

Comment:
It is better to change engineType to codeType

Reply:
> It is better to change engineType to codeType

Thanks @guoshupei  ,have updated variable name to avoid misunderstanding.